app/views/layouts/_nav-admin.html.erb

```diff
@@ -0,0 +1,15 @@
+<nav>
```

Comment:
@alischlereth @iandouglas Most of admin-specific rights live in the nav bar. Because of this, I am not doing very much namespacing. Default users and Admin users both visit '/gifs' to see all gifs, but admins have the ability to add / remove gifs. This is built into the nav and one if statement in the gifs index to create a delete category button if the user is an admin. Is this an okay way to approach this?

Reply:
Yes, it's common for a nav view to have logic like that. Here's how I'd do it:
```ruby
<% if current_admin %>
  user is logged in and an admin
<% elsif current_user %>
  user is logged in
<% else %>
  user is anonymous
<% end %>
```